frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx

```diff
@@ -47,6 +47,10 @@ interface IIdentificationProps {
   handlePinChange: (pin: string, nameSpace?: string) => void;
   /** whether or not this user is an admin and should have greater access to fields */
   isAdmin: boolean;
+  /** whether or not this form is being displayed as part of a view or update */
+  isViewOrUpdate: boolean;
```

Comment:
What's the other view option?  Is it 'Add'?

Reply:
yeah, this form supports three different display modes:
1) View or update with the form disabled
2) View or update with the form enabled
3) Add from scratch